config/eslintrc.yaml

```diff
@@ -158,3 +160,13 @@ rules:
   jsdoc/require-hyphen-before-param-description: warn
   jsdoc/check-indentation: warn
   jsdoc/require-description-complete-sentence: warn
+
+# Overwrite jest's expect rules for cypress test files, which uses a different
```

Comment:
does this require `.editorconfig` adjustments as well?

Reply:
No